A few changelog tweaks

amenonsen opened the pull request:
This corrects a problem where the "facts on delegated hosts" item was inserted between the `meta: refresh_inventory` item and its description and fixes the wording in a few other places.

amenonsen committed:
A few changelog tweaks

amenonsen committed:
Mention ansible_ssh_pipelining too

bcoca commented on CHANGELOG.md:
```diff
@@ -302,12 +302,13 @@ newline being stripped you can change your playbook like this:
-* The undocumented use of ';' as a separator is now deprecated.
+* We now recommend `pattern1,pattern2` to combine host matching patterns.
+  The use of ':' as a separator conflicts with IPv6 addresses and host ranges. It will be deprecated in the future.
+* The undocumented use of ';' as a separator is no longer supported.
```
it is supported, it is now deprecated

amenonsen committed:
We support ; as a deprecated pattern separator again

(But we shouldn't recommend ':' as a replacement, when it's going to be
deprecated as well.)

amenonsen commented:
Oh, I didn't notice the code had changed. I've updated the PR.
